### ISSUE TITLE
error deploying web app

### ISSUE THREAD

**Issue by chrisdias** [Microsoft/vscode-azureappservice#529]
Repro steps:
* started vs code insiders
* opened my app
* right click on the explorer and choose "deploy to web app..."
* waited for somethign to happen (login i assume)
* opened azure explorer and continued to wait
* got a drop down for "Select a Web App"

result:

![image](https://user-images.githubusercontent.com/1487073/42789226-de82c534-8918-11e8-82c6-57f726179622.png)

Action: appService.Deploy
Error type: object
Error Message: {"key":"1","value":{"_creator":null,"_nextState":null,"_state":{"name":"error","done":null,"then":null},"_value":{},"_listeners":null,"_isException":false,"_errorId":2}}

Version: 0.9.0
OS: win32

**Comment by nturinski** [comment]
Hi @chrisdias,

Thanks for reporting this issue.  Have you been able to consistently repro this?  I've been trying to do the exact same thing, but it just prompts me to deploy as expected.

**Comment by chrisdias** [comment]
no, i can't repro :(.  i had a very cold start and it was taking quite some time for log-in, i suspect there was a timing issue because of that.
